subsys/partition_manager/CMakeLists.txt

```diff
@@ -73,6 +73,10 @@ if (CONFIG_BUILD_WITH_TFM)
   ncs_add_partition_manager_config(pm.yml.tfm)
 endif()
 
+if (CONFIG_PM_RESERVE_HUK_KEY)
+  ncs_add_partition_manager_config(pm.yml.huk_nrf52840)
```

Comment:
It would be preferable to have this call in  `lib/huk_load/CMakeLists.txt`

 I understand why you put it here (since there already is a lot of stuff here) but ideally all of this should be spread out to where it really belongs :)

Reply:
I think that my previous comment apply here. Let me quote: 
> I think that we need this flexibility to flash the key. The CONFIG_HUK_LOAD is enabled in b0 and it will fail booting if it cannot load the HUK. So we need this for applications to be able to flash the HUK. Am I missing something here?

Reply:
I'm not sure I understand. My suggestion is simply to move this piece of code to another file, it will be executed under the same condition (CONFIG_PM_RESERVE_HUK_KEY).

Reply:
With the current implementation this will not work, right? 
https://github.com/nrfconnect/sdk-nrf/blob/21352ec7b9c72eb00d13bdac75571c9b588fb68b/lib/CMakeLists.txt#L31

I can of course change the change this so that it requires the PM_RESERVE_... but that sounds a bit misleading, isn't it?

Reply:
Ok, so if it is possible that you want this partition without having set `CONFIG_HW_UNIQUE_KEY_LOAD` then I have misunderstood and you can ignore this proposal :)

Reply:
I will change that ;)